Content.Client/Recycling/RecyclerVisualizerSystem.cs

```diff
@@ -0,0 +1,59 @@
+using Content.Shared.Conveyor;
+using Content.Shared.Recycling;
+using Robust.Client.GameObjects;
+
+namespace Content.Client.Recycling
+{
+    public sealed class RecyclerVisualizerSystem : VisualizerSystem<RecyclerComponent>
```

Comment:
Im think this one can be made into a `GenericVisualizer`, so take a look at them (still gonna leave the other reviews in case it isnt possible, but if you do this you can obviously disregard them)